queries/reports/filters.js

```diff
@@ -91,25 +91,16 @@ export function governmentDomains(params) {
     if (!governmentDomains) {
       return ``;
     }
-    // This is a temporary hack for reports with an end date before 2022-03-02, which is the first date of a publication with a normalized policy domain
-    // this fix should be removed after the poicy domains have been normalized
-    let publicationDateRange = params.filter.publicationDate;
-    if (publicationDateRange) {
-      const [publicationDateStart, publicationDateEnd] = publicationDateRange;
-      if (publicationDateEnd && publicationDateEnd < new Date('2022-03-02')) {
-        return ``;
-      }
-    }
-
     let _governmentDomains = governmentDomains.map((uri) => sparqlEscapeUri(uri));
     return `
 {
   SELECT DISTINCT ?publicationFlow WHERE {
     VALUES ?governmentDomain { ${ _governmentDomains.join('\n') } }
     GRAPH <http://mu.semte.ch/graphs/organizations/kanselarij> {
       ?publicationFlow dossier:behandelt ?case .
-      ?case a dossier:Dossier ;
-        besluitvorming:beleidsveld ?governmentDomain .
+      ?case a dossier:Dossier .
+      ?case dossier:Dossier.isNeerslagVan ?decisionmakingFlow .
+      ?decisionmakingFlow besluitvorming:beleidsveld ?governmentDomain .
```

Comment:
I think this will now only return reports for the publications "via ministerraad", but we also need the others.

This should use the governmentAreas attached to the `publicationFlow` directly, not those attached to the `decisionmakingFlow`. These should be synced up after the migrations I added last week.

So using `?publicationFlow pub:beleidsveld ?governmentDomain .` should suffice. (just make sure `PREFIX pub: <http://mu.semte.ch/vocabularies/ext/publicatie/>` is set)